packages/pci/pci.0.2.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1"
```

Comment:
you might need to use "1.2" as a version here. If you run `opam lint` in the directory where these files are, it will tell you better :p